bioconda_utils/utils.py

```diff
@@ -50,11 +55,24 @@ def emit(self, record):
 
 
 def tqdm(*args, **kwargs):
-    """Wrapper around TQDM handling disable"""
-    enable = (sys.stderr.isatty()
-              and os.environ.get("TERM", "") != "dumb"
-              and os.environ.get("CIRCLECI", "") != "true")
-    kwargs['disable'] = not enable
+    """Wrapper around TQDM handling disable
+
+    Logging is disabled if:
+
+    - ``TERM`` is set to ``dumb``
+    - ``CIRCLECI`` is set to ``true``
+    - the effective log level of the is lower than set via ``loglevel``
+
+    Args:
+      loglevel: logging loglevel (the number, so logging.INFO)
+      logger: local logger (in case it has different effective log level)
+    """
+    term_ok = (sys.stderr.isatty()
+               and os.environ.get("TERM", "") != "dumb"
+               and os.environ.get("CIRCLECI", "") != "true")
+    loglevel_ok = (kwargs.get('logger', logger).getEffectiveLevel()
+                   <= kwargs.get('loglevel', logging.INFO))
```

Comment:
@dpryan79 - This makes the progress monitor now silent if you pass `--loglevel warn`.